katalog/prometheus-operated/deploy.yml

```diff
@@ -31,10 +31,7 @@ spec:
       cpu: 2000m
       memory: 2Gi
   retention: 30d
-  ruleSelector:
-    matchLabels:
-      prometheus: k8s
-      role: alert-rules
+  ruleSelector: {}
```

Comment:
Why we are removing this?

Reply:
@angelbarrera92 I removed it to be consistent with the upstream. This [PR](https://github.com/prometheus-operator/kube-prometheus/pull/1315) removes it.